src/markdown-pages/contribute-to-quickstarts/instrument-library/send-events.mdx

```diff
@@ -0,0 +1,170 @@
+
+```bash
+$ pip install newrelic-telemetry-sdk
+```
+For Java, you will need to set up several dependencies:
+```java
+//Maven Dependencies
+    <dependency>
+      <groupId>com.newrelic.telemetry</groupId>
+      <artifactId>telemetry-core</artifactId>
+      <version>0.13.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.newrelic.telemetry</groupId>
+      <artifactId>telemetry-http-okhttp</artifactId>
+      <version>0.13.1</version>
+    </dependency>
+
+ //Gradle Dependencies
+    implementation("com.newrelic.telemetry:telemetry-core:0.13.1")
+    implementation("com.newrelic.telemetry:telemetry-http-okhttp:0.13.1")
+```
+
+Once our SDK is installed and ready, you can use the library to easily set up the api call to New Relic. 
+
+* **Python**
+```Python
+import os
+import time
+from newrelic_telemetry_sdk import Event, EventClient
+
+# Record that a rate limit has been applied to an endpoint for an account
+event = Event(
+    "RateLimit", {"path": "/v1/endpoint", "accountId": 1000, "rejectRatio": 0.1}
+)
+
+event_client = EventClient(os.environ["NEW_RELIC_INSERT_KEY"])
+response = event_client.send(event)
+response.raise_for_status()
+print("Event sent successfully!")
+
+```
+* **Java**
+```java
+//Imports
+import static java.util.Collections.singleton;
+
+import com.newrelic.telemetry.Attributes;
+import com.newrelic.telemetry.OkHttpPoster;
+import com.newrelic.telemetry.TelemetryClient;
+import com.newrelic.telemetry.events.Event;
+import com.newrelic.telemetry.events.EventBatch;
+import java.net.InetAddress;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.UUID;
+
+public class TelemetryClientExample {
+
+  public static void main(String[] args) throws Exception {
+    String insightsInsertKey = args[0];
+
+    // create a TelemetryClient with an http connect timeout of 10 seconds.
+    TelemetryClient telemetryClient =
+        TelemetryClient.create(
+            () -> new OkHttpPoster(Duration.of(10, ChronoUnit.SECONDS)), insightsInsertKey);
+    Attributes commonAttributes =
+        new Attributes()
+            .put("exampleName", "TelemetryClientExample")
+            .put("service.name", "TelemetryClientExampleService")
+            .put("host.hostname", InetAddress.getLocalHost().getHostName())
+            .put("environment", "staging");
+
+
+    Span span = sendSampleEvent(telemetryClient, commonAttributes);
+
+    // make sure to shutdown the client, else the background Executor will stop the program from
+    // exiting.
+    telemetryClient.shutdown();
+  }
+
+  private static void sendSampleEvent(
+      TelemetryClient telemetryClient, Attributes commonAttributes) {
+    Event event = new Event("TestEvent", new Attributes().put("testKey", "testValue"));
+    telemetryClient.sendBatch(new EventBatch(singleton(event), commonAttributes));
+  }
+
+```
+### Alternative Options
```

Comment:
One option you haven't covered is just making API requests with the developer's favorite http library (in other words, manual instrumentation). Is this not something you see partners doing?

Reply:
That's true, I guess I should just direct them to our event API page in that case?

Reply:
I added something for this.